docker-app/qfieldcloud/core/tests/test_api.py

```diff
@@ -36,9 +36,23 @@ def setUp(self):
 
     def test_api_status(self):
         response = self.client.get("/api/v1/status/")
+
         self.assertTrue(status.is_success(response.status_code))
-        self.assertEqual(response.json()["storage"], "ok")
```

Comment:
Can we actually do it a bit more backwards compatible:

`storage` is a global status - if all storages are ok, then it is ok, if any of the storages has error, then it is error.

And we keep the new key "storages"?